### PR TITLE
Add a drink! Shembot

### PR DESCRIPTION
**Pull request by joamarderije**
This is my add my drink to your list, One of the best drinks that Filipinos proud of, One of a kind mix you will surely like

### CODE REVIEW

File: src/recipes/shembot.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "1 Bottle of Ginebra San Miguel ",
+        "1 25g Pack of Melon Juice Flavor",
+        "1 25g pack of Coffee",
+        "a pack of cube ice"
+    ],
+    "directions": [
+        "1.get a pitcher filled with water(3/4).",
+        "2.pour the Gin into the pitcher together with the melon juice and nescafe.",
+        "3.after pouring the gin in the pitcher, light the bottle using a match or a lighter and subside bottle in the pitcher",
+        "4. serve it with ice ;)"
```
Comment (alfg) [inline src/recipes/shembot.json]: Hi, the numeration can be removed as this list is rendered as an ordered list and will display numbers already.

**Comment by joamarderije** [comment]
Hi I updated the changes needed for this pull request

**Comment by joamarderije** [comment]
Already fix your request :D

File: src/recipes/shembot.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            "quantity": "1",
+            "measure": "25g/pack",
+            "ingredient": "Any Brand of Brown Coffee"
+        },
+        {
+            "quantity": "1",
+            "measure": "pack",
+            "ingredient": "Cube Ice"
+        },
```
Comment (alfg) [inline src/recipes/shembot.json]: Remove comma here.
```suggestion
        }
```

Comment (joamarderije) [reply src/recipes/shembot.json]: done removing the comma in line 25

**Pull request merged**